pkg/systemd/renderer.js

```diff
@@ -35,6 +35,19 @@ define([
 
     var journal_renderer = {};
 
+    function make_printable(value) {
+        if (value === undefined)
+            return _("[no data]");
+        else if ($.type(value) == "string")
+            return value;
+        else if (value.length !== undefined)
+            return cockpit.format(_("[$0 bytes of unprintable data]"), value.length);
```

Comment:
Is `unprintable` a good term for this? What about binary / raw?
@andreasn What do you think?